routers/users.py

```
@@ -0,0 +1,143 @@
+    token = jwt.encode(payload, settings.SECRET_KEY, algorithm=settings.ALGORITHM)
+    expires_in = settings.ACCESS_TOKEN_EXPIRE_MINUTES * 60
+    return token, expires_in
+
+
+
+DB_FILE = "users.json"
+
+def get_db_users():
+    if not os.path.exists(DB_FILE):
+        return []
+    with open(DB_FILE, "r", encoding="utf-8") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)